README.md

```diff
@@ -19,4 +19,8 @@
  Gradle task. For more information please see the official Jetbrains plugin
  development documentation. 
  
- http://www.jetbrains.org/intellij/sdk/docs/tutorials/build_system.html
\ No newline at end of file
+ http://www.jetbrains.org/intellij/sdk/docs/tutorials/build_system.html
+ 
+ 
+ ## Plugin in the JetBrains site
```

Comment:
Thanks for the contribution @AVATOR! If you don't mind changing "in" to "on" I'll merge this right in!

`Plugin in the JetBrains site` to `Plugin on the JetBrains site`